server/src/packages/packages.controller.ts

```diff
@@ -174,6 +181,7 @@ import { APPLICANT_ATTRS } from './pas-form/applicants/applicants.attrs';
                 dcp_applicantinformationid: `representative-${applicant.dcp_applicantrepresentativeinformationid}`
               }}),
             ],
+            bbls: landuseForm.dcp_dcp_projectbbl_dcp_landuse,
```

Comment:
What are some ways to make this branching logic more organized? Maybe each form's service owns how it constructs the object?

Reply:
Interesting idea. Is it possible to activate a Serializer on a controller that isn't the endpoint getting called? I can look into it...